website/content/configuration/_index.md

```diff
@@ -265,6 +265,24 @@ Addresses can still be specifically allocated from the "expensive"
 pool with the methods described in
 the [usage](/usage/#requesting-specific-ips) section.
 
+### Single-IP address pool
+
+As the implementation uses `net.ParseCIDR`, you need to provide a /32 mask for a single IP pool.
```

Comment:
I wouldn't mention implementation details in the user-facing docs, but rather simply state what's supported without the "why".
I would also omit the YAML snippet since all we have to do is mention that entries under `addresses` should always contain a mask, even when the CIDR is `/32`.

Reply:
You're right @johananl , whole section might be overkill for this.

As a single IP pool make sense only when dealing with multiple address pool & controlling automatic address allocation, I choose this section to update.

I added another commit rather than squashing, so approver still have the possibility to revert to the previous proposition.